homework4/4_2.py

```diff
@@ -0,0 +1,95 @@
+"""Task 2."""
+
+
+class Bar(object):
+    """Class Bar."""
+
+    client_count = 0
+
+    def __init__(self, name, age):
+        """Attributes."""
+        self.name = name
+        self.age = age
+        self.total_price = 0
+        self.food = []
+        self.alcohol = []
+        Bar.client_count += 1
+
+    def display_count(self):
+        """Count."""
+        print('Total number of visits %d' % Bar.client_count)
+
+    def display_visit(self):
```

Comment:
I don't want bar to display my visit. It's a good idea if my wife could access this information :)